### ISSUE TITLE
replace phpmyadmin for adminer

### ISSUE THREAD

**Issue by Darkspac3** [UNIT3D/UNIT3D-Installer#1]
For me Adminer is a better alternative to replace phpMyAdmin one file, easy, ect... 
[
See detailed comparison](https://www.adminer.org/en/phpmyadmin/)

**Comment by HDVinnie** [comment]
I personally dislike Adminer but maybe will make it optional to choose phpmyadmin or Adminer.

**Comment by Darkspac3** [comment]
Why you dislike Adminer?

**Comment by HDVinnie** [comment]
Put it this way. I personally prefer phpmyadmin. As I said though I will make it optional

**Comment by Darkspac3** [comment]
Ok thanks you Can you add more : 
- Mail server
- DNS server, auto-config domains with nginx and https with lets encrypt auto renewal
- FTPS
- Add optional MariaDB if it is work with unit3d

Good Work ^^

**Comment by HDVinnie** [comment]
Yes a lot more will be added........this is more a script to use to setup your server with all the perquisites needed to run UNIT3D. So pretty much one can install Ubuntu on there server then run this shell script (once finished) to install all the goodies needed and clone UNIT3D.
